server/src/test/java/e2e/FailureHandlingTest.java

```diff
@@ -185,7 +198,9 @@ public Workflow handleClientSpecificException() {
         return new WorkflowImpl("handle-client-specific-exception", thread -> {
             NodeOutput node = thread.execute("business-exception-failure");
             thread.handleException(node, handler -> {
-                handler.execute("my-handler");
+                WfRunVariable failureContent =
+                        handler.addVariable(WorkflowThread.HANDLER_INPUT_VAR, VariableType.JSON_OBJ);
```

Comment:
Perfect, this e2e is exactly what we need. Only thing left to do is update the documentation :smile: